index.html

```diff
@@ -10,17 +10,26 @@
   <meta name="msapplication-config" content="/static/favicons/browserconfig.xml">
   <meta name="theme-color" content="#ffffff">
 
+  <meta name="description" content="Tranport One est une entreprise de transport pour personnes agées, handicapées, adultes ou enfants basée à Genève.">
+  <meta name="keywords" content="transport,transport one,genève,vtc,chauffeur">
```

Comment:
Actually keywords are sometimes treated as spammy things and can be penalized ^^

Anyway just for your info, let it like this for now!